src/parser/jobs/rpr/modules/DISPLAY_ORDER.ts

```diff
@@ -1,6 +1,13 @@
 export const DISPLAY_ORDER = {
-	IMMORTAL_SACRIFICE: 10,
-	ARCANE_CIRCLE: 15,
-	INTERRUPTS: 20,
-	ENSHROUD: 90,
+	IMMORTAL_SACRIFICE: 1,
+	ARCANE_CIRCLE: 2,
+	ENSHROUD: 3,
+
+	// Super low priority, some of these override default ordering
+	AOE: 50,
+	COMBOS: 51,
+	POSITIONALS: 52,
+	WEAVING: 53,
+	INTERRUPTS: 54,
+	TINCTURES: 55,
```

Comment:
Nit; I'd put Tinctures under the rest of Reaper unique logic as we have expected actions which makes it something of note to check.

Arcane Circle > Enshroud > Tinctures > Positional >weaving issues > combo issues > Incorrect AoE Action Usage is the order I'd want to check. but perhaps keeping the "super low priority" ones in the same order as other jobs for consistency would be better.

Reply:
This probably needs more standardisation in core, module titles as well. Gonna leave it for now even if it's less consistent.